src/store/groups.js

```diff
@@ -22,6 +22,24 @@
 
 const state = {
 	groups: [],
+	sortedGroups: [],
+	orderGroups: 'amount',
+}
+
+// the alphabetical sort is the default sort one so its undefined
+const sortFunc = {
+	amount: (groupA, groupB) => groupB.contacts.length - groupA.contacts.length,
+}
+const sortGroups = state => {
+	state.sortedGroups = state.groups
+		.map(group => {
+			/* .toString is given for the default .sort() method
+				in which every object is converted to string and their utf-16 code is compared
+				*/
+			group.toString = () => group.name
```

Comment:
is the comment good enough for understanding or should i go for
```js
{...group, toString: () => group.name}
// or
Object.assign(group, { toString: group.name })
```

Reply:
No the comment is enough I'd say.
Though you can use the long js comment vor clarity if you prefer :)

```js
			/*
			 * .toString is given for the default .sort() method
			 * in which every object is converted to string and their utf-16 code is compared
			 */
```

Reply:
eh, good enough, can change for jsdoc if thats what you meant